cirq_superstaq/ops/qubit_gates.py

```diff
@@ -753,6 +753,75 @@ def _from_json_dict_(cls, **kwargs: Any) -> "IXGate":
 AQTICCX = AQTITOFFOLI = IX.controlled(2, [0, 0])
 
 
+@cirq.value_equality(approximate=True)
+class StrippedCZGate(cirq.Gate):
+    """The Stripped CZ Gate is the gate that is actually being performed by Hilbert,
+    which is just a regular CZ gate when phi = 0, and is corrected into a CZ
+    gate by RZ gates afterwards if phi is nonzero.
+    """
+
+    def __init__(self, phi: cirq.TParamVal = 0) -> None:
+        """Args: phi: CZ-rotation angle in radians"""
+        self.phi = phi
```

Comment:
nit
```suggestion
        self._phi = phi

    @property
    def phi(self) -> cirq.TParamVal:
        return self._phi
```
(or something else in place of `phi`, as per prior comment)